setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='numpy_stubs',
+    name='numpy-stubs',
```

Comment:
I see that PEP 561 changed it's guidelines here. I suppose we should also rename this repository?

Reply:
This name (the distribution name) doesn't impact PEP 561 compliance. The *package name* is the real decider; when this repo starts using my PEP 561 implementation in mypy, it will need to rename the `numpy` package to `numpy-stubs`, and make according changes. Happy to do those changes when it is time.